views/albums.hbs

```diff
@@ -0,0 +1,10 @@
+<hi>In albums</hi>
+<ul>
```

Comment:
Everything works fine, the only issue here is how you're nesting other elements than <li> inside the <ul>.

You should only have <li> items, you can nest other elements within it like this for instance
```
<ul>
    {{#each albums}}
    <li> 
     <div>
       <span>{{this.name}}</span>
       <br> 
       <img src="{{this.images.1.url}}" alt="{{this.name}}">     
       <a href="/tracks/{{this.id}}"><button type="button"View tracks</button></a>
       {{/each}}
       <br>
     </div>
  </li>
</ul>